content/python/concepts/built-in-functions/terms/super/super.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: 'super()'
+Description: 'implicitly refer to the superclass'
```

Comment:
```suggestion
Description: 'Returns a temporary object that allows a given class to inherit the methods and properties of a parent or sibling class.'
```

Let's make sure to use capitalization and punctuation. 😄

Reply:
Updated